src/Xrefcheck/Core.hs

```diff
@@ -102,9 +106,9 @@ data FileInfoDiff = FileInfoDiff
   }
 makeLenses ''FileInfoDiff
 
-diffToFileInfo :: FileInfoDiff -> FileInfo
-diffToFileInfo (FileInfoDiff refs anchors) =
-    FileInfo (DList.toList refs) (DList.toList anchors)
+diffToFileInfo :: Bool -> FileInfoDiff -> FileInfo
+diffToFileInfo ignoreCpcInFile (FileInfoDiff refs anchors) =
```

Comment:
`ignoreCpcInFile` stands for copy paste check enabled or disabled? It is being used directly as the `_fiCopyPasteCheck` field argument.

Reply:
Yes, it's a typo